vagen/envs/webarena/llms/providers/api_utils.py

```
@@ -0,0 +1,711 @@
+                try:
+                    # Load the credentials from the key file
+                    creds = service_account.Credentials.from_service_account_file(
+                        GCLOUD_KEY_FILE_PATH,
+                        # You can list multiple scopes if needed
+                        scopes=['https://www.googleapis.com/auth/cloud-platform']  
+                    )
+                    # Refresh the token (this is needed even for the first time)
+                    creds.refresh(Request())
+                    return creds.token
+
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)